runtest.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Can we avoid having this script, or at least, avoid putting it in the root directory? It looks like it basically just runs a `diff` command? Some options:

1) Include it directly in the meson.build files (I'm not sure what's possible with meson).
2) Put it at `tests/runtest.sh`, and reference it from `examples/meson.build`.
3) Copy it both places (it's tiny).